src/branchManagement.ts

```diff
@@ -10,8 +10,22 @@ import { getBranchNameSeparator, getPrefixes, getValidateBranchNameWhiteList } f
  */
 export async function getBranchDetails(): Promise<{ branchNameFinal: string; approval: string } | null> {
     const branchNameSeparator = vscode.workspace.getConfiguration('branch-creator').get<string>('branchNameSeparator', '-');
+    const apps = vscode.workspace.getConfiguration('branch-creator').get<string[]>('appsList', []);
+    const appFirst = vscode.workspace.getConfiguration('branch-creator').get<boolean>('appFirst', false);
+    let appName = '';
+    if (appFirst && apps.length > 0) {
```

Comment:
it's not a big deal but because you compare apps.length> 0 several time but you only initialise  apps once you can declare a variable called 
isAppsNotEmpty = app.length > 0